### PR TITLE
Appium Hands Off Feedback

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Description
Added a caution regarding JDK and Maven dependencies before running sample test.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/automated-testing/appium/quickstart.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The following list of prerequisites are specific to running tests written in Jav
 * A [GitHub](https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home) account
 * [Git](https://git-scm.com/downloads)
 * The [Swag Labs iOS Real Device Sample App](https://github.com/saucelabs/sample-app-mobile/releases)
-* [Java Testing Dependencies](https://training.saucelabs.com/codelabs/Module1-SeleniumJava/index.html?index=..%2F..SeleniumJava#4): Java, Maven, IDE (such as IntelliJ)
+* Local installation of JDK, Maven, and and IDE such as IntelliJ. For instructions on installing eachof these, see the [Java Testing Dependencies](https://training.saucelabs.com/codelabs/Module1-SeleniumJava/index.html?index=..%2F..SeleniumJava#4) training module.
```
Comment (lysannep) [inline docs/mobile-apps/automated-testing/appium/quickstart.md:27]:
```suggestion
* Local installation of JDK, Maven, and an IDE such as IntelliJ. For instructions on installing each of these, see the [Java Testing Dependencies](https://training.saucelabs.com/codelabs/Module1-SeleniumJava/index.html?index=..%2F..SeleniumJava#4) training module.
```

**Comment by sweeneyskirt-sl** [comment]
You rule, @lysannep -- thanks for cleaning up my mess. Can you tell I am rushing to get all my stuff checked in before IT shut my laptop down! LOL

File: docs/mobile-apps/automated-testing/appium/quickstart.md [hunk 2]
```diff
@@ -66,7 +66,18 @@ Set your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` as environment variables to avo
...
+```
+## Maven Variables
+export M2_HOME=/Applications/apache-maven-3.8.2
+export M2=$M2_HOME/bin
+
+export PATH=$PATH:$M2:$JAVA_HOME/bin:$PATH
+```
+:::
+
+If you are using an IDE, such as IntelliJ or Visual Studio Code, open the `quickstart-appium-java` project you cloned in step 2, then right-click the `src/test/java/tests/RDC/AppiumIosRdcAppTest` and choose `Run`.
```
Comment (lysannep) [inline docs/mobile-apps/automated-testing/appium/quickstart.md:80]: These instructions are specific to IntelliJ - the steps to run the test are a bit different for Visual Studio Code. 
```suggestion
If you are using an IDE, such as IntelliJ, open the `quickstart-appium-java` project you cloned in step 2, then right-click the `src/test/java/tests/RDC/AppiumIosRdcAppTest` and choose `Run`.
```

**Review by lysannep — APPROVED** [review]
Looks good - thanks Nancy!

**Pull request merged**